terraform/datadog/datadog.tf

```diff
@@ -0,0 +1,29 @@
+variable "datadog_api_key" {}
+variable "datadog_app_key" {}
+variable "env" {}
+
+provider "datadog" {
+    api_key = "${var.datadog_api_key}"
+    app_key = "${var.datadog_app_key}"
+}
+
+resource "datadog_monitor" "router" {
+  name = "${format("%s router hosts", var.env)}"
+  type = "service check"
+  message = "Missing router hosts in environment {{host.environment}}. Notify: @the-multi-cloud-paas-team@digital.cabinet-office.gov.uk"
```

Comment:
http://docs.datadoghq.com/monitoring/#notifications seems to suggest that the `Notify: ` is not needed - `@target-email` in the message is enough to trigger notification. Perhaps a more complete sentence would be nice, if you are making a change here anyway.